source/crud/crud.rst

```diff
@@ -1762,6 +1762,7 @@ Q: Where is ``singleBatch`` in FindOptions?
 Changes
 =======
 
+* 2019-05-01: Specify a document or pipeline for commands with updates.
```

Comment:
Here, and possibly someplace else, it should be documented that the update can only be a pipeline with MongoDB 4.2+.

Reply:
I have added this in the change comment.